src/vscode-dts/vscode.proposed.remoteCodingAgents.d.ts

```diff
@@ -3,6 +3,27 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-// empty placeholder for coding agent contribution point from core
+declare module 'vscode' {
+	export interface RemoteCodingAgentInformation {
+		id: number;
+		number: number;
+		title: string;
+		user: {
+			login: string;
+		};
+		html_url: string;
+		state: string;
+		created_at: string;
+		updated_at: string;
+	}
 
-// @joshspicer
+	export interface RemoteCodingAgentInformationProvider extends Disposable {
+		onDidChangeCodingAgentInformation: Event<RemoteCodingAgentInformation>;
+		onDidSelectItem: (codingAgentId: string) => void;
```

Comment:
What will we do when the user select an item? We generally try to avoid leaking UI concepts directly into providers so I don't think we want `onDidSelectItem`